getting-started/getting-started.adoc

```diff
@@ -0,0 +1,49 @@
+= Getting Started with Eclipse MicroProfile
+
+The MicroProfile is a specification and does not provide a reference implementation. Many vendors already support MicroProfile and any other vendor is free to implement MicroProfile, or a specific API from MicroProfile.
```

Comment:
Yes, perhaps the "does not provide a reference implementation" bit should be moved to the bit below where implementations are listed?